platform/golden/upload-image.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# Check for or create 'debian-golden' bucket in OCI
+if [ $(oci os bucket list | jq '.data[0].name' | grep "debian-golden" | wc -l) -ge 1 ] ; then
+  echo "bucket debian-golden exists"
+else
+  oci os bucket create --name debian-golden
+  if [ $? -gt 0 ]; then exit 1; fi
+fi
+
+# Check for and upload golden.raw disk image to 'debian-golden' bucket in OCI
+if [ -f build/golden.raw-disk001.vmdk ]; then
+  oci os object put --bucket-name debian-golden --file build/golden.raw-disk001.vmdk --name golden.raw-disk001.vmdk --force
+  if [ $? -gt 0 ]; then exit 1; fi
+else
+  echo no vmdk to upload!
+  exit 1
+fi
+
+# Set up and track the import of the image into OCI
+NAMESPACE=$(oci os ns get | jq .data | tr -d '"')
+WORK_REQUEST=$(oci compute image import from-object --namespace $NAMESPACE \
+	--launch-mode PARAVIRTUALIZED --display-name debian-golden \
```

Comment:
PARAVIRTUALIZED mode, as defined by Oracle, is superior. I thought to launch in `PARAVIRTUALIZED` required EFI, rather than BIOS.

Nice!